papaya-web/build.gradle

```diff
@@ -31,8 +31,7 @@ dependencies {
     compile project(':papaya-core')
     compile 'com.google.guava:guava:27.1-jre'
     annotationProcessor 'org.immutables:value:2.7.5'
-    implementation group: 'com.github.papaya-agh', name: 'papaya-interface', version: 'master-SNAPSHOT',
-            changing: true
+    implementation group: 'com.github.papaya-agh', name: 'papaya-interface', version: 'IO2019TEAM-66-SNAPSHOT'
```

Comment:
To `changing` powinno sprawiać, że zależność zawsze Ci się zaciąga od nowa zamiast bycia pokonanym przez cache Gradle'a przy amendach w `papaya-interface`

Reply:
W sumie to miałem jakieś problemy, żeby się to pobierało, więc odświeżam ręcznie.  I tak w tym pliku nie będzie żadnych zmian po wprowadzeniu api na mastera